### ISSUE TITLE
A stable version of the plugin is shown as experimental in the plugin manager

### ISSUE THREAD

**Issue by ptormene** [gem/oq-irmt-qgis#346]
![screen shot 2017-11-28 at 14 48 00](https://user-images.githubusercontent.com/350930/33328482-65d8793a-d45a-11e7-9298-2a9bf16a114b.png)

As in the screenshot, although version 2.3.7 is not checked as "experimental", the plugin manager states it is. In this case, a newer experimental version exists, but "show experimental plugins" is unchecked. I guess it's a bug of QGIS.

**Comment by mbernasocchi** [comment]
are you sure you dont have maybe a symlinked folder in the installation? how does it look like on the plugin webpage with the versions?

**Comment by ptormene** [comment]
![screenshot from 2017-11-29 09-52-50](https://user-images.githubusercontent.com/350930/33366318-29db4c8c-d4eb-11e7-8e01-6f5aa1ddab74.png)

It was noticed during the risk workshop, and Cata says it happens on every computer. I am quite sure they don't have a symlinked folder in the installation.

**Comment by daniviga** [comment]
I was there, and I can confirm. I was pretty surprised to see 2.7.3 as experimental.

@ptormene 2.7.3 was experimental before and then released as stable or has been pushed directly to stable?

It could be that they had old metadata in cache from the days before.

**Comment by ptormene** [comment]
Yes, I usually keep the flag "experimental" in the plugin metadata when I make a new release, and I mark the version as "stable" using the web GUI once the scientists have confirmed it works on their machines.

**Comment by mbernasocchi** [comment]
mmm maybe thats the issue?
